lambdas/endpoint/src/email/get-owner.ts

```diff
@@ -0,0 +1,46 @@
+import type { APIGatewayProxyEvent } from 'aws-lambda';
+import {
+  CognitoIdentityProviderClient,
+  GetUserCommand,
+} from '@aws-sdk/client-cognito-identity-provider';
+import { ErrorWithStatusCode } from '../error-with-status-code';
+
+const cognitoClient = new CognitoIdentityProviderClient({
+  region: 'eu-west-2',
+});
+
+export const getOwner = async (
+  event: APIGatewayProxyEvent
+): Promise<{
+  username: string;
+  emailAddress: string;
+}> => {
+  const accessToken = event.headers.Authorization;
+
+  if (!accessToken) {
+    throw new ErrorWithStatusCode('Missing access token', 400);
+  }
+
+  const { Username, UserAttributes } = await cognitoClient.send(
```

Comment:
There's a change I requested in Ben's PR (#197) where you can pass this stuff into the handler lambda from the authorizer lambda as request context, and then pull it from `event.requestContext.authorizer`. Ben has it passing the username down, but we could add the email address on too - it would mean that we don't have to hit cognito here. We're already doing this same `GetUserCommand` in the authorizer, so we can just reuse the result from that.

Reply:
Although the bulk of the work to support this has been done in Ben's PR, it might be best to merge this first and refactor it in that PR.